City-of-Helsinki/helsinki-paatokset
# PP-240: Policymaker pages

To test: 
- `make fresh` or `drush cim`  && `drush cr`
- Add Helsinki Kanava credentials here: https://helsinki-paatokset.docker.so/fi/admin/config/helsinki-kanava/settings (get them here: https://helsinkisolutionoffice.atlassian.net/wiki/spaces/PP/pages/2689106104/Helsinki-kanavan+videot)
- Run migrations: `drush mim paatokset_meeting_videos`

Visual fixes to check (https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto)
- Video embed should now have a cover image instead of just black helsinki logo on grey
- Ensure that embedded meeting record is the most recent one
- All recordings -link should take you to the most recent video on Helsinki Kanava page
- h2- and section element margins should match the sketch
- All of these links work:
![image](https://user-images.githubusercontent.com/49063836/141807306-9426b762-dd9a-483d-a4af-b1da10d34138.png)
- Try adding image to the page via the `policymaker_image` -field. It should be displayed with decent resolution and not be grainy
- Try adding content to newly added fields (`field_documents_description`, `field_recording_description`, `field_meetings_description` and `field_decisions_description`). They should look in line with the sketch. Note that `field_documents_description` is also displayed on `/asiakirjat` -page. `field_decisions_description` is only displayed on policymakers of type trustee, e.g. https://helsinki-paatokset.docker.so/fi/paattajat/pormestari

Other things to check:
- Adding a node and giving it a path alias like `/paattajat/*` should not cause the node view to crash
